pages/Announcements/Types/Tombstone.md

```diff
@@ -0,0 +1,65 @@
+---
+name: "Type: Tombstone"
+route: /Announcements/Types/Tombstone
+menu: Announcements
+---
+
+# Tombstone Announcement
+
+A Tombstone Announcement is a way to note that a previous announcement signature is invalid
+and the related Announcement should be considered reverted.
```

Comment:
One sentence per line

Reply:
I thought the rule was max one sentence per line, thus splitting up a sentence was ok for readability. Thoughts?